eng/pipelines/common/variables.yml

```diff
@@ -12,9 +12,9 @@ variables:
 - name: isOfficialBuild
   value: ${{ and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}
 - name: isFullMatrix
-  value: ${{ and(eq(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}
+  value: ${{ or(in(variables['Build.DefinitionName'], 'manual-runtime-staging', 'manual-runtime'), and(eq(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest'))) }}
```

Comment:
I would be careful of adding a pipeline that runs with `isfullMatrix=true` for `runtime.yml` as that burns a lot of resources and runs a ton of stuff (it is equivalent to our rolling builds which we just reduced to 2 a day), so maybe we can be more granular there?

Reply:
yeah, I think it can go in another variable.